src/window.rs

```diff
@@ -822,6 +822,16 @@ impl Window {
         self.window.set_minimized(minimized);
     }
 
+    /// Gets the window's current minimized state.
+    ///
+    /// ## Platform-specific
```

Comment:
While I think that `minimize` makes sense, could you return `Option<bool>` instead? On Wayland for example you can request minimize, but you don't know the state, whether you're minimized or not.

The impl for that on Wayland will be plain `None` return.

On other platforms you could also return `None` while they are un-implemented, meaning that you can't determine whether the window is minimized or not.